index.html

```diff
@@ -11,13 +11,14 @@
   <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
   <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+  <script defer src="index.js"></script>
 </head>
 <body>
-  <main>
+  <main data-main>
     <header class="flex">
       <a href="#" class="logo">My Logo</a>
       <nav>
-        <div id="menu-btn">
+        <div id="menu-btn" data-menu>
           <div class="menu-bar"></div>
           <div class="menu-bar"></div>
           <div class="menu-bar"></div>
```

Comment:
- [OPTIONAL] Kindly make sure that the links in the navbar on the desktop view are working as well and redirect the user to the right section of the page.